public/js/roles/roles-list.controller.js

```diff
@@ -64,5 +65,30 @@
                 vm.getRoles(vm.page);
             });
         }
+
+        function deleteRole (roleId) {
+            api.getUsers().then(function (res) {
```

Comment:
You should limit the fields returned by `/api/users` to avoid fetching & parsing all users data.
Since you only uses the `roles` property, you can do `api.getUsers({fields: 'roles'}).then(...)`

Reply:
I agree it's changed